specification/monitor/resource-manager/Microsoft.Insights/stable/2021-05-01/metrics_API.json

```diff
@@ -291,7 +291,7 @@
           "format": "duration"
         },
         "metricNames": {
-          "description": "The names of the metrics (comma separated) to retrieve.",
```

Comment:
For api versions > 2019-07-01, a comma within the metric name needs to be double encoded, otherwise it will count as two separate metrics. "%252C"